gtda/externals/bindings/simplex_tree_bindings.cpp

```diff
@@ -59,6 +60,8 @@ PYBIND11_MODULE(gtda_simplex_tree, m) {
   py::class_<Persistent_cohomology_interface_inst>(
       m, "Simplex_tree_persistence_interface")
       .def(py::init<simplex_tree_interface_inst*, bool>())
+      .def("compute_persistence",
```

Comment:
As above.

Reply:
We can add it, I mean if it useful later, why not have it ? :)